arbeitszeiterfassung_s4a/arbeitszeiterfassung_s4a/doctype/working_time/working_time.py

```diff
@@ -7,12 +7,11 @@
 from frappe import _
 from frappe.model.document import Document
 
-HALF_DAY = 3.25
-OVERTIME_FACTOR = 1.15
-MAX_HALF_DAY = HALF_DAY * OVERTIME_FACTOR * 60 * 60
 FIVE_MINUTES = 5 * 60
 ONE_HOUR = 60 * 60
 
+def get_default_activity():
+	return frappe.db.get_single_value("Working Time Settings", "default_activity")
```

Comment:
```suggestion
    return frappe.db.get_single_value("Working Time Settings", "default_activity")
```

Der Rest der Datei nutzt vier Leerzeichen für die Einrückung